get-started/part3.md

```diff
@@ -103,6 +103,19 @@ This `docker-compose.yml` file tells Docker to do the following:
 - Define the `webnet` network with the default settings (which is a
   load-balanced overlay network).
 
+Notice that we've set the Compose file version to version 3. This essentially
+makes it [swarm mode](/engine/swarm/index.md) compatible. We can make use of the
```

Comment:
Totally agree in keeping complexity low, since this is supposed to be your first interaction with Docker, so at this point I know nothing about compose files, docker-compose, or docker stack deploy.

So this might not be the best place to explain me the difference between all these things, because the answer is going to be complex, and  distract me from deploying my first app.